src/Entity.php

```diff
@@ -414,9 +426,76 @@ public function prepareInputForUpdate($input)
             $input = $this->checkRightDatas($input);
         }
 
+        $input = $this->handleCustomScenesInputValues($input);
+
+        return $input;
+    }
+
+    private function handleCustomScenesInputValues(array $input): array
+    {
+        $fields = [
+            "custom_helpdesk_home_scene_left",
+            "custom_helpdesk_home_scene_right",
+        ];
+
+        foreach ($fields as $field) {
+            // Check for special values
+            $value = $input[$field] ?? "";
+            if (empty($value)) {
+                continue;
+            }
+
+            if ($value == self::SCENE_DEFAULT) {
+                // Reset default value (empty string)
+                $input[$field] = "";
```

Comment:
Maybe storing `default` in DB instead of an empty string would be easier to read.

Reply:
This is not ideal because the field is name "custom_scene_left/right".
An empty value make sense to mean there are no custom scene defined.

If you want to be extra precise, we need an extra field that store the behavior : default/custom/-2.
This would be clearer but come at the cost of adding 2 extra fields to a table that already have 124 columns.

I'm not against it, just let me know what you want.